src/Orleans/CodeGeneration/GrainFactoryBase.cs

```diff
@@ -157,6 +157,21 @@ public static class GrainFactoryBase
                     grainClassNamePrefix);
         }
 
+        internal static IAddressable MakeGrainReference_FromType(
```

Comment:
This is just an exact copy from GrainFactory._MakeGrainReference. Just moved it here.